force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -291,8 +386,26 @@ public with sharing class VariablesDataTypesOperators {
      * @return A Decimal representing the monthly paycheck.
      */
     public static Decimal calculateMonthlyPaycheck(Decimal hourlyRate, Double numberOfHours) {
-        return null; // Replace null with the variable you used to store the result
+        final Integer NUMBER_OF_WEEKLY_PAYCHECKS = 4;
+        Decimal monthlyPaycheck;
+        // Ensure valid input: hourlyRate should be > 0, numberOfHours should be positive
+        if (hourlyRate <= 0 || numberOfHours <= 0) {
+            return monthlyPaycheck = 0.0; // Return 0.0 if invalid input
+        }
+    
+        // Calculate the weekly paycheck first
+        Decimal weeklyPaycheck = calculateWeeklyPaycheck(hourlyRate, numberOfHours);
+    
+        // Check if weekly paycheck is 0, return 0.0 if true
+        if (weeklyPaycheck == 0.0) {
+            return monthlyPaycheck = 0.0;
+        }
+        // Calculate the monthly paycheck using the formula: weekly paycheck multiplied by 4
+        monthlyPaycheck = weeklyPaycheck * NUMBER_OF_WEEKLY_PAYCHECKS;
+        return monthlyPaycheck; // Return the calculated monthly paycheck
```

Comment:
See if there's a way you can call the method above calculateWeeklyPaycheck to do this work for you.